roles/variables/defaults/main.yml

```diff
@@ -1578,7 +1578,6 @@ kafka_connect_replicator_status_topic: connect-status
 ### Set this variable to customize the topic where Kafka Connect Replicator stores it's configuration.
 kafka_connect_replicator_storage_topic: connect-configs
 
-### Set this variable to customize the topic where Kafka Connect Replicator consumer stores it's timestamps.
```

Comment:
We can have ## and modify the comment to explain where we are using this variable.
That'll ensure it won't go to variables.md and still have some context if someone stumbles into this.

Reply:
makes sense @utkarsh5474 .. I was just afraid someone accidentally adding third # making it configurable again :)